.rubocop_todo.yml

```diff
@@ -422,6 +422,7 @@ Style/DateTime:
     - 'app/models/conference.rb'
     - 'app/models/program.rb'
     - 'spec/models/conference_spec.rb'
+    - 'spec/models/program_spec.rb'
```

Comment:
I would expect your commit message to say a tiny bit more as to why you added this here.